packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -0,0 +1,155 @@
+import { Input } from '@botonic/core'
+import axios from 'axios'
+
+import {
+  HtFallbackNode,
+  HtFlowBuilderData,
+  HtIntentNode,
+  HtKeywordNode,
+  HtNodeComponent,
+  HtNodeStartType,
+  HtNodeWithContent,
+  HtNodeWithContentType,
+  HtStartNode,
+} from './content-fields/hubtype-fields'
+import { FlowBuilderApiOptions } from './types'
+
+export class FlowBuilderApi {
+  url: string
+  flow: HtFlowBuilderData
+
+  private constructor() {}
+
+  static async create(options: FlowBuilderApiOptions): Promise<FlowBuilderApi> {
+    const newApi = new FlowBuilderApi()
+
+    newApi.url = options.url
+    newApi.flow = options.flow ?? (await newApi.getFlow(options.accessToken))
```

Comment:
I would create a method that given the object options, gets the flow, or directly (much better) passing both flow and access token

Reply:
Sorry Marc this is from another PR I didn't put the base branch correctly. Now it is correct. 
options.flow is the case where the flow is in a local json (defined inside the bot). If this is not defined, this is when the call is actually made with the access token.